extra-admin/grub/autobuild/patches/10009-Strip-all-strings-before-kernel-version-number.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/10_linux.orig b/10_linux
+old mode 100644
+new mode 100755
+index b82ecee..e79af9c
+--- a/10_linux.orig
```

Comment:
Actually, there is an issue here. The file edited should be correspondent to the file in the GRUB source file. See the newest 10008 patch I edited just yesterday.